src/components/LazyLog/index.jsx

```diff
@@ -377,16 +382,23 @@ export default class LazyLog extends Component {
 
   handleSearch = keywords => {
     const { resultLines, searchKeywords } = this.state;
+    let inputKeywords = keywords;
+
+    if (this.props.caseInsensitive) {
+      inputKeywords = inputKeywords.toLowerCase();
+      this.encodedLog = encode(decode(this.encodedLog).toLowerCase());
```

Comment:
I feel like changing `this.encodedLog` for the LazyLog component is not the most ideal way. Perhaps we can do that in `searchLines` by having that function another parameter:

```jsx
export const searchLines = (rawKeywords, rawLog, isCaseInsensitive) => {
  // ...
};
```